app/views/content_items/statistical_data_set.html.erb

```diff
@@ -1,7 +1,41 @@
-<%= render 'shared/title_and_translations', content_item: @content_item %>
-<%= render 'components/notice', @content_item.withdrawal_notice_component  %>
-<%= render 'shared/metadata', content_item: @content_item %>
-<%= render 'shared/history_notice', content_item: @content_item %>
-<%= render 'govuk_component/lead_paragraph', text: @content_item.description %>
-<%= render 'shared/sidebar_contents_with_body', content_item: @content_item %>
-<%= render 'shared/footer', @content_item.document_footer %>
+<%= content_for :simple_header, @navigation.should_present_taxonomy_navigation? %>
```

Comment:
I'm not sure we should be applying taxonomy navigation conditions to this format. It doesn't look like there was a previous taxonomy nav variant for this format so we should either add this to all formats or maintain the previous behaviour i.e. don't set `:simple_header` based on taxonomy navigation applicability.

Reply:
As @steventux says, this will always be false for statistical data sets:
https://github.com/alphagov/government-frontend/blob/d42d4056c8325c90b06ea89e56557addbb457730/app/models/navigation_type.rb#L9-L13

Reply:
Shall I remove this completely then? 
Does it need replacing with anything?

Reply:
Yes remove completely - the navigation is always present in the current live version, it should behave the same way.